docker/dockerfiles/Dockerfile_opensource_image.template

```diff
@@ -50,6 +50,7 @@ ENV NOTEBOOKS_VENV ${OPENVINO_WORKBENCH_DATA_PATH}/jupyter_env
 
 ENV DEPENDENCIES " \
     unzip \
+    git \
```

Comment:
Why do you need git inside docker?

Reply:
This requirement require git https://github.com/openvinotoolkit/workbench/blob/master/requirements/requirements.txt#L12

Reply:
👍